New f: 9
New f: 10
New f: 11
New f: 12
terminate called after throwing an instance of 'St9bad_alloc'
  what():  std::bad_alloc
